src/htcdaskgateway/cluster.py

```diff
@@ -33,7 +44,7 @@ async def _stop_async(self):
 
         self.status = "closed"
     
-    def scale(self, n, **kwargs):
+    def scale(self, n):
```

Comment:
@Nanoemc why were the **kwargs removed from this method?

Reply:
I realized it didn't do anything, you would have to pass in kwargs when the user did cluster.scale(10). I thought you needed it to access the kwargs in gateway.new_cluster(), but that doesn't seem to work, I was just accessing stuff already stored in self. I don't think a user would need kwargs for scale, but maybe in the future?

Reply:
It's in the [original code](https://github.com/dask/dask-gateway/blob/bf04d65def1c6d6ca758d14b849735e2368789c8/dask-gateway/dask_gateway/client.py#L1091) so if it doesn't make any difference we should keep it as it was before (with **kwargs)

Reply:
Ah ok, I thought I had added it so I will add it back